### DISCUSSION TITLE
Uncaught ReferenceError: process is not defined

### DISCUSSION THREAD

**Discussion by knightswatch3** [webpack/webpack#12978] [Q&A]
I do have this piece of code confirming the value of process.env.NODE_ENV :

`new webpack.DefinePlugin({
      "process.env": {
        NODE_ENV: process.env.NODE_ENV ? "'" + process.env.NODE_ENV + "'" : null,
      }
    })`

But I receive this following error in the browser. 

**`Uncaught ReferenceError: process is not defined`**

I tried configuring it via dotenv but that's not helping me.

**Comment by alexander-akait** [comment]
Because you have library/code which use `process`, it is not safe for browser, but you can polyfill it using `window.process`
